### PR TITLE
Winston' hw

### PR DESCRIPTION
**Pull request by zorba-strange**
CL_5
C_4 did not nest my resources. get distracted by tmux

### CODE REVIEW

File: game/app/assets/stylesheets/application.css [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+ *
+ * You're free to add application-wide styles to this file and they'll appear at the bottom of the
+ * compiled file so the styles you add here take precedence over styles defined in any other CSS/SCSS
+ * files in this directory. Styles in this file should be added after the last require_* statement.
+ * It is generally better to create a new file per style scope.
+ *
+ *= require_tree .
+ *= require_self
+ */
+
```
Comment (superbuggy) [inline game/app/assets/stylesheets/application.css:16]: Love the styling here!

File: game/app/controllers/characters_controller.rb [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  end
+
+    
+
+  private
+  def character_params
+    params.require(:character).permit(:name, :photo_url, :house_id)
+  end
+
+end
```
Comment (superbuggy) [inline game/app/controllers/characters_controller.rb:54]: Controller looks solid!

File: game/app/controllers/houses_controller.rb [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    @house.destroy
+    redirect_to houses_path, notice: "#{@house.name} Was Eliminated!"
+  end
+
+  private
+  def house_params
+    params.require(:house).permit(:name, :rulers_of_the, :photo_url)
+  end
+
+
```
Comment (superbuggy) [inline game/app/controllers/houses_controller.rb:49]: This controller looks good as well. Creative flash notices!

File: game/app/views/characters/_form.html.erb [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+<%= form_for @character do |f| %>
+  <%= f.label :name %>
+  <%= f.text_field :name %>
+
+  <%= f.label :photo_url %>
+  <%= f.text_field :photo_url %>
+
+  <%= f.label :house_id %>
+  <%= f.select :house_id, options_for_select(@house.map{ |x| [x.name, x.id]}) %>
```
Comment (superbuggy) [inline game/app/views/characters/_form.html.erb:15]: 👍  noice

File: game/app/views/houses/show.html.erb [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+    data: {confirm: "Are You Sure?"}%></h3>
+  <h3 class="border_box"><%= link_to "Add Family Member", new_character_path %></h3>
+  <h3 class="border_box"><%= link_to "Update", edit_house_path(@house.id) %></h3>
+</div>
+<% @character.each do |char| %>
+  <div class="char_image">
+    <h2 class="char_name"><%=link_to char.name, character_path(char.id), :class=>"char_name" %></h2>
+    <img src="<%= char.photo_url %>">
+  </div>
+<% end %>
```
Comment (superbuggy) [inline game/app/views/houses/show.html.erb:17]: 👍

File: game/app/views/layouts/application.html.erb [hunk 6]
```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Game of Thrones</title>
+    <%= csrf_meta_tags %>
+
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.0/jquery.min.js"></script>
+    <!-- Latest compiled and minified CSS -->
```
Comment (superbuggy) [inline game/app/views/layouts/application.html.erb:8]: why jQuery?

File: game/db/migrate/20160801182446_create_houses.rb [hunk 7]
```diff
@@ -0,0 +1,12 @@
...
+    create_table :houses do |t|
+      t.string :name
+      t.string :rulers_of_the
+      t.string :from
+      t.string :photo_url
+
+      t.timestamps
+    end
+  end
+end
```
Comment (superbuggy) [inline game/db/migrate/20160801182446_create_houses.rb:12]: 👍  though `rulers_of_the` might be a little unclear or non-semantic

File: game/db/migrate/20160801182504_create_characters.rb [hunk 8]
```diff
@@ -0,0 +1,11 @@
+class CreateCharacters < ActiveRecord::Migration[5.0]
+  def change
+    create_table :characters do |t|
+      t.string  :name
+      t.string  :photo_url
+      t.references  :house, index: true, foreign_key: true
+
```
Comment (superbuggy) [inline game/db/migrate/20160801182504_create_characters.rb:7]: awesome

File: game/db/seeds.rb [hunk 9]
```diff
@@ -0,0 +1,18 @@
...
+  from: "Eyrie", 
+  photo_url:"http://vignette3.wikia.nocookie.net/gameofthrones/images/6/62/House-Arryn-heraldry.jpg/revision/latest?cb=20140402114006"
+}) 
+
+characters = Character.create!({
+  name: "Jon Arryn", 
+  photo_url: "http://nerdist.com/wp-content/uploads/2016/01/John-Standing-Game-of-Thrones.jpg", 
+  house: current_house
+})
+
```
Comment (superbuggy) [inline game/db/seeds.rb:17]: solid seed file; think about adding more however

**Comment by superbuggy** [comment]
Good work on this lab! See comments within.
